feat(body): add entities formatting support (#250)
EdJoPaTo/grammy-inline-menu: README.md

});
```

> ### Can I use `entities` like with `grammyjs/parse-mode` in the message body?
> 
> Yes, you can pass the option `entities`.

Reviewer: Thought about linking to the Repo like the example above also links to `telegram-format`. Also, the current sentence is kinda self explaining so it can be removed in my opinion. What do you think?

```suggestion
### Can I use `entities` in the message body?

Also see: [`grammyjs/parse-mode`](https://github.com/grammyjs/parse-mode)
```

```ts
import { bold, fmt, underline } from '@grammyjs/parse-mode';

const menu = new MenuTemplate<MyContext>(async () => {

> 	const message = fmt`${bold(underline('Hello world!'))}`

Reviewer: ```suggestion
	const message = fmt`${bold(underline('Hello world!'))}`;
```

});
```

> ### Can I use entities or `grammyjs/parse-mode` in the message body?

Reviewer: ```suggestion
### Can I use `entities` like with `grammyjs/parse-mode` in the message body?
```

Yes, you can pass the option `entities`.

> This will be resolved automatically to`caption_entities` if you send a `media` message.

Reviewer: I think I would remove this statement. It's meant to be transparently handled and is the same for `text` and `caption` so I don't think there is much confusion about it.
(But I know the library well, so maybe it's helpful for others?)

```ts

> const menu = new MenuTemplate<MyContext>(async () => {

Reviewer: Can you add the import here to clarify it's a third party example?

	const message = fmt`${bold(underline('Hello world!'))}`

> 

Reviewer: Removing the empty line seems more concise to me.

```suggestion
```

Author: Thanks a lot for your feedback, I updated the documentation
